insights/client/utilities.py

```diff
@@ -235,7 +242,10 @@ def magic_plan_b(filename):
 
 def generate_container_id(container_name):
     # container id is a uuid in the namespace of the machine
-    return str(uuid.uuid5(uuid.UUID(generate_machine_id()), container_name.encode('utf8')))
```

Comment:
IIRC per our discussion earlier this week, we do want containers specifically to take the host into consideration when generating IDs but not images.  Does the `analyze_image` option get used when scanning containers?